src/devices_models/device_constructors/HydroReservoirCascade_constructor.jl

```diff
@@ -19,16 +18,6 @@ function PSI.construct_device!(
     #Initial Conditions
     PSI.storage_energy_init(psi_container, devices)
 
-    #Constraints
```

Comment:
Why are these constraints deleted?

Reply:
Since the active power variables are only constrained by the `active_power limits` and not a time series, we don't need to add extra constraints.  The bounds on the variable are sufficient.